clang/lib/Frontend/InitPreprocessor.cpp

```diff
@@ -740,6 +748,8 @@ static void InitializeCPlusPlusFeatureTestMacros(const LangOptions &LangOpts,
   if (LangOpts.Char8)
     Builder.defineMacro("__cpp_char8_t", "202207L");
   Builder.defineMacro("__cpp_impl_destroying_delete", "201806L");
+
+  Builder.defineMacro("__cpp_pp_embed", "202403L");
```

Comment:
I wonder if for Clang18 if we're comfortable (given the short runway) claiming full support?

Reply:
Good catch -- this should be removed entirely. We don't want to define a C++ feature test macro until WG21 adopts the feature and finalizes the macro name and value.